### ISSUE TITLE
Page loads with divs displayed over images...

### ISSUE THREAD

**Issue by mwhaynes** [codrops/DirectionAwareHoverEffect#7]
I'm sure there's something simple that I've done in setting up the Direction Aware Hover Effect for a test site. When the page first loads, the overlay divs appear over the images until I mouse over them, then they disappear as I mouse over each image. From that point everything works as expected. Has anybody else seen this? How do I fix it? Does it have something to do with the container div that the unordered list is placed in? 

Thanks for any insights!

**Comment by hamzadhamiya** [comment]
Same issue was with me, so I fixed it by adding left:-100% to the div.

**Comment by jkiss** [comment]
yeah, there is no method to init this, so i make a new branch [DirectionAwareHoverEffect](https://github.com/jkiss/DirectionAwareHoverEffect)

**Comment by Ynk33** [comment]
same problem here
